mbta/routes_config.go

```diff
@@ -0,0 +1,80 @@
+package mbta
+
+import "net/http"
+
+// GetAllRoutesSortByType all of the possible ways to sort by for a GetAllRoutes request
+type GetAllRoutesSortByType string
+
+const (
+	GetAllRoutesSortByColorAscending                 GetAllRoutesSortByType = "color"
+	GetAllRoutesSortByColorDescending                GetAllRoutesSortByType = "-color"
+	GetAllRoutesSortByDescriptionAscending           GetAllRoutesSortByType = "description"
+	GetAllRoutesSortByDescriptionDescending          GetAllRoutesSortByType = "-description"
+	GetAllRoutesSortByDirectionDestinationAscending  GetAllRoutesSortByType = "direction_destinations"
+	GetAllRoutesSortByDirectionDestinationDescending GetAllRoutesSortByType = "-direction_destinations"
+	GetAllRoutesSortByDirectionNameAscending         GetAllRoutesSortByType = "direction_names"
+	GetAllRoutesSortByDirectionNameDescending        GetAllRoutesSortByType = "-direction_names"
+	GetAllRoutesSortByFareClassAscending             GetAllRoutesSortByType = "fare_class"
+	GetAllRoutesSortByFareClassDescending            GetAllRoutesSortByType = "-fare_class"
+	GetAllRoutesSortByLongNameDesending              GetAllRoutesSortByType = "long_name"
+	GetAllRoutesSortByLongNameDescending             GetAllRoutesSortByType = "-long_name"
+	GetAllRoutesSortByShortNameAscending             GetAllRoutesSortByType = "short_name"
+	GetAllRoutesSortByShortNameDescending            GetAllRoutesSortByType = "-short_name"
+	GetAllRoutesSortBySortOrderAscending             GetAllRoutesSortByType = "sort_order"
+	GetAllRoutesSortBySortOrderDescending            GetAllRoutesSortByType = "-sort_order"
+	GetAllRoutesSortByTextColorAscending             GetAllRoutesSortByType = "text_color"
+	GetAllRoutesSortByTextColorDescending            GetAllRoutesSortByType = "-text_color"
+	GetAllRoutesSortByTypeAscending                  GetAllRoutesSortByType = "type"
+	GetAllRoutesSortByTypeDescending                 GetAllRoutesSortByType = "-type"
+)
+
+// GetAllRoutesRequestConfig extra options for the GetAllRoutes request
+type GetAllRoutesRequestConfig struct {
+	PageOffset        string                 // Offset (0-based) of first element in the page
+	PageLimit         string                 // Max number of elements to return
+	Sort              GetAllRoutesSortByType // Results can be sorted by the id or any GetAllRoutesSortType
+	IncludeStop       bool                   // Include Stop data is request
```

Comment:
This `IncludeStop` is never used. It should be used in the `includes` statement but that request have an option for `IncludeStop`